### PR TITLE
Add options to put method

### PR DESCRIPTION
**Pull request by EmanH**
See: https://developers.cloudflare.com/workers/kv/expiring-keys/#creating-expiring-keys-from-a-worker

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -108,7 +108,15 @@ declare module '@cloudflare/workers-types' {
...
     get(key: string, type?: 'text' | 'json' | 'arrayBuffer' | 'stream'): Promise<any>
 
-    put(key: string, value: string | ReadableStream | ArrayBuffer | FormData): Promise<void>
+    put(
+      key: string, 
+      value: string | ReadableStream | ArrayBuffer | FormData,
+      options?: { 
+        expiration?: string | number;
+        expirationTtl?: string | number;
+        [propName: string]: any;
```
Comment (jrf0110) [inline index.d.ts]: Why the index signature? This would allow a key like `expirationTTL`

**Comment by jrf0110** [comment]
Thanks for the PR. Looks great other than my comment

File: index.d.ts (hunk 1 above)
Comment (EmanH) [reply index.d.ts]: Problematic, true. To account for unknown options? I couldn't find comprehensive Worker KV API docs, so wasn't sure if there are other options.

I guess better to leave that out and add options as we're aware of them. Yes?

**Comment by jrf0110** [comment]
Published in `1.0.4`

**Pull request merged**